Cargo.toml

```diff
@@ -41,3 +41,6 @@ toml = "0.5.9"
 [dev-dependencies]
 assert_cmd = "2.0"
 lazy_static = "1.4.0"
+
+[features]
+vendored-openssl = ["crates-index/vendored-openssl"]
```

Comment:
Were you still planning on making this feature be default-enabled to preserve the current behavior, or did you change your mind on it again?

Reply:
Oh sorry, I forgot to make a commit for making it a default feature. Gimme a sec.

Reply:
Thanks for the remainder. adcf9ef9d08f5c6109c16695327a68a6528c3efe